pomegranate/NaiveBayes.pyx

```diff
@@ -58,23 +58,40 @@ cdef class NaiveBayes( object ):
            [-0.26751248, -1.4493653 ],
            [-1.09861229, -0.40546511]])
     """
-
+    
     cdef int initialized
     cdef public object models
     cdef void** models_ptr
     cdef numpy.ndarray summaries
     cdef public numpy.ndarray weights
     cdef double* weights_ptr
+    # dimension of inputs, 0 is dimensionless, -1 if uninitialized
+    cdef public int d
```

Comment:
Since this has a defined meaning in all other classes, it should be consistent, with d meaning the number of dimensions the model considers.